src/BuildScriptGenerator/DotNetCore/DotNetCoreBashBuildSnippet.sh.tpl

```diff
@@ -9,17 +9,25 @@ echo "Using .NET Core SDK Version: $dotnetCoreVersion"
     {{ InstallBlazorWebAssemblyAOTWorkloadCommand }}
 {{ end }}
 
+doc="https://docs.microsoft.com/en-us/azure/app-service/configure-language-dotnetcore?pivots=platform-linux"
+suggestion="Please build your app locally before publishing." 
+msg="${suggestion} | ${doc}"
+
 {{ # .NET Core 1.1 based projects require restore to be run before publish }}
-dotnet restore "{{ ProjectFile }}"
+cmd="dotnet restore \"{{ ProjectFile }}\""
+LogErrorWithTryCatch "$cmd" "$msg"
```

Comment:
What's the benefit we're gaining here from adding the custom logging message before exiting with the same error code? Is the output from the failed command not currently being propagated to the user in the output logs?

Reply:
It does get propagated. However our team is looking for custom logging (fields delimited by `|`) so it can be parsed by the a parser that propagates this to our users in a readable format.

Reply:
Project Spotlight Documentation: https://microsoft-my.sharepoint.com/:w:/p/jafreebe/EXuT-xXGvS5BrcInLq5iZ_oB8NbWtupREAAayzwSc7cP5w?e=mYqNNV&nav=eyJjIjoxNjM3MjMwOTAwfQ